package.json

```diff
@@ -54,19 +54,19 @@
     "lodash.findindex": "4.6.0",
     "lodash.isequal": "4.5.0",
     "lodash.template": "4.5.0",
-    "mini-css-extract-plugin": "^0.9.0",
+    "mini-css-extract-plugin": "^1.4.0",
     "multi-loader": "0.1.0",
     "powerbi-visuals-webpack-plugin": "^2.2.5",
```

Comment:
Use the latest powerbi-visuals-webpack-plugin v2.3.0

Reply:
implemented latest version 2.2.6